### DISCUSSION TITLE
发帖求助付了款收不到验证码用不了专业版，留了凭证一直没有解决，解决不了麻烦退款！

### DISCUSSION THREAD

**Discussion by juzhier** [getlantern/lantern-forum-cn#818] [General]
8月8日买的专业版，支付账号邮箱leecx23184260@gmail.com ，这个邮箱一直收不到验证码，很麻烦的是即用蓝灯，gmail邮箱打开也很艰难。我有发帖求助，但是让我提供相关凭证之后再也没有回复，到目前为止我依然是免费版的状态，无法使用专业版。如果你们解决不了问题，麻烦给我退款！
![Uploading 微信图片_20240813083611.jpg…]()

**Comment by MaxMKMax** [comment]
@juzhier 您好. 非常抱歉影响了您的体验感. 我马上给您找人解决问题. 然后给您补偿 2 个星期的时间. 您可以发有支付单据到我的邮箱吗? 这里被屏蔽了. 我看不到您的支付单据. 谢谢.
mike@getlantern.org

**Reply by juzhier** [reply]
我今天已经发送邮件到你的邮箱，麻烦看到帮我尽快解决专业版的问题。因为给你们留言也经常没有二次回复，而且之前我发过一次邮箱，并没有人理睬。
